### ISSUE TITLE
Setup Windows for GitHub Actions

### ISSUE THREAD

**Issue by faustbrian** [ArkEcosystem/cpp-client#130]
https://github.com/ArkEcosystem/cpp-client/commit/a188803b7cbc67931010be0b645c4b7c1343eff6 moved everything CI related to GitHub Actions. Missing is windows to ditch appveyor which is a bit unreliable.

**Comment by faustbrian** [comment]
@ciband @sleepdefic1t could you look into what configurations you need to get the tests and build working on GitHub Actions with windows? Should be fairly simple like the rest of the workflows.

**Comment by sleepdefic1t** [comment]
Definitely not as straight forward as the others. lol
I've got it triggering and building in a fork branch atm, but not finding the test .exe that it _supposedly_ made..

I'll try to get it wrapped up this week.

**Comment by sleepdefic1t** [comment]
Battled this on and off for a while.
Still not much progress.

tl;dr
issues building/linking to cURL on Windows.

If any Windows devs would like to take a look, that could be helpful.
I'm not too familiar with that env.

Another solution would be to re-add Hunter for package management,
but it'd only be for handling cURL.
C++ Client & Crypto otherwise just use Git via CMake.

**Comment by sleepdefic1t** [comment]
resolved in #204
